src/Kunstmaan/FormBundle/Event/SubmissionEvent.php

```diff
@@ -3,6 +3,7 @@
 namespace Kunstmaan\FormBundle\Event;
 
 use Kunstmaan\FormBundle\Entity\FormSubmission;
+use Kunstmaan\FormBundle\Helper\FormPageInterface;
 use Kunstmaan\NodeBundle\Entity\AbstractPage;
```

Comment:
Can you switch out all of the AbstractPage typehints because this is an unused use statement now.